index.ts

```diff
@@ -10,6 +10,7 @@ addTests('__urls_that_dont_match__', [
 	'https://github.com/sindresorhus/refined-github/issues/new',
 	'https://github.com/sindresorhus/refined-github/issues/new/choose',
 	'https://github.com/sindresorhus/refined-github/issues/templates/edit',
+	'https://github.com/sindresorhus/refined-github?search=1',
```

Comment:
Not sure it's the right place to add it.

Feel free to comment if you have suggestions!

Reply:
Yep this is the right place

Reply:
Wait, no, this URL is already matched by `isRepoTree`. I think you meant:
```suggestion
	'https://github.com?search=1',
```

Reply:
> I think you meant

Actually, no. But move to `isDashboard` is better.

I forgot to delete `https://github.com/sindresorhus/refined-github?search=1` from `__urls_that_dont_match__` when I added `https://github.com/sindresorhus/refined-github?search=1` to `isRepoTree`